src/components/Booking/EventTypeGroup/index.js

```diff
@@ -1,23 +1,33 @@
 import React, { Component } from 'react';
 import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { PropTypes as PT } from 'prop-types';
 import { Icon } from 'react-native-elements';
 import { GREY, WHITE } from '../../../styles/colors';
+import type from '../../../constants/eventTypes';
 
 class EventTypeGroup extends Component {
+  static propTypes = {
```

Comment:
not a huge deal, but do we have a standard place to put proptypes? closer to the bottom of the file?

Reply:
we only put propTypes at bottom of file if it is a stateless class

Reply:
cool, thats fair